File: admin/src/app/models/create-requests/bill-detail-create-request.ts
import { BaseCreateRequest } from "@app/models";
import { AutoMap } from "@automapper/classes";
export class BillDetailCreateRequest extends BaseCreateRequest {
    @AutoMap()
    productId: number;
    @AutoMap()
    price: number;
    @AutoMap()
    quantity: number;
    @AutoMap()
    discountPrice: number;
}

File: App.Models/DTOs/BillDetails/BillDetailCreateRequest.cs
using App.Models.DTOs.CreateRequests;
using App.Common.Model.DTOs;

namespace App.Models.DTOs.BillDetails
{
    public class BillDetailCreateRequest : BaseCreateRequest
    {
        public int ProductId { get; set; }
        public int? Quantity { get; set; }
        public double? Price { get; set; }
        public double? DiscountPrice { get; set; }
    }
}
